support-files/mariadb-execstartpre-check.sh

```diff
@@ -0,0 +1,66 @@
+#!/bin/bash
+#
+# This script purpose is to check if MariaDB
+# configuration has problematic configurations
+# and make more reasonable error messages than
+# it does now
+#
+# To make this script work you need:
+#   * my_print_defaults from MariaDB package
+#   * systemctl from systemd package
+#
+MY_PRINT_DEFAULTS=$(which my_print_defaults)
```

Comment:
As this script is going to go through CMake preprocessor

`if test -x @bindir@/my_print_defaults` can fill out the install path according to distribution layout. Same `@bindir@` for execution.

If you want a fallback plan, like for bintar packages, you can use a relative path from `$(dirname $0)`.

Reply:
Ok i'll update CMakelist.txt and make it work as expected.